paket.template

```diff
@@ -0,0 +1,24 @@
+type file
+id ScriptCs.FSharp
+version 0.2.0
+summary A ScriptCs script engine for F#
+description
+    A ScriptCs script engine for F#
+owners
+    panesofglass
+authors
+    Glenn Block
+    Dave Thomas
+    Ryan Riley
+    Filip W
+language en-US
+projectUrl https://github.com/scriptcs-contrib/scriptcs-fsharp
+licenseUrl https://raw.githubusercontent.com/scriptcs-contrib/scriptcs-fsharp/master/LICENSE.txt
+files
```

Comment:
This should be handled automatically if you change to ``type project`` - just remove the file list completely. If it doesn't do the right thing at that point, it's probably a paket pack bug.